pkg/operator/status/controller.go

```diff
@@ -79,6 +80,8 @@ func (c StatusSyncer) sync() error {
 	if err != nil && !apierrors.IsNotFound(err) {
 		return err
 	}
+	originalConfig := operatorConfig.DeepCopy()
```

Comment:
realizing I started this pattern and we actually want the copy to be the one we modify.